runtime/runtime_test.go

```diff
@@ -409,22 +409,37 @@ func (i *testRuntimeInterface) Hash(data []byte, tag string, hashAlgorithm HashA
 }
 
 func (i *testRuntimeInterface) SetCadenceValue(owner common.Address, key string, value cadence.Value) (err error) {
+	if i.setCadenceValue == nil {
+		panic("must specify testRuntimeInterface.setCadenceValue")
```

Comment:
Good idea! Maybe even do this consistently for all functions, currently some just return a default of no hook is provided. Better to have to explicitly provide functions that are called in the tests

Reply:
Ah I just switched some to having defaults! I can switch them all over though. I agree that being explicit is better since defaults tend to hide bugs.

Reply:
Agreed, better to switch to `panic`s, it's a great idea!

Reply:
A LOT of tests fail when I eliminate the defaults. I created https://github.com/onflow/cadence/issues/1400 so we can account for the work and maybe assign it later depending on priority. It's a constrained enough problem that I should be able to it without needing much if any help but I don't want to push more urgent stuff back.

However, there are a few more methods that currently panic with a generic message instead of a nice one. I'm going to add them to this PR before asking for another review (hopefully super quick) then merging into master.

Reply:
Sounds good! Thank you for trying and opening the separate PR 👍